topi/python/topi/intel_graphics/conv2d.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=invalid-name,unused-variable,unused-argument,no-else-return, too-many-arguments, too-many-locals, too-many-statements, no-member, too-many-branches
+# pylint: disable=invalid-name,unused-variable,unused-argument,no-else-return, too-many-arguments, too-many-locals, too-many-statements, no-member, too-many-branches, too-many-boolean-expressions
 """conv2d schedule on Intel Graphics"""
 
 from __future__ import absolute_import as _abs
```

Comment:
given that we have the attributes support in compute already, is it possible to move the hint information into the attributes?

Reply:
By hint information, do you mean the tags such as 2_14, 1_16? And for the attributes support, is it in cfg?

Reply:
I see, the performance decreases about 8% for end-to-end performance when moving the conv_tags to attributes.

Reply:
in theory they should be the same? do you mean the compilation time?

Reply:
No, it's the runtime. I feel it weird as well. So I tried both opt_level = 2 and opt_level = 3. But both gives be performance drop. But layerized convolution performance doesn't drop.